Cargo.toml

```diff
@@ -67,3 +67,5 @@ name = "r1cs"
 harness = false
 required-features = ["yoloproofs", "avx2_backend"]
 
+[patch.crates-io]
+curve25519-dalek = { git = "https://github.com/remoun/curve25519-dalek.git", rev = "77391036a6e43f3c9a2d8b937e621dcfe9d95d8c" }
```

Comment:
this should be a mobilecoin hosted fork

Reply:
Agreed! Can I get your review on https://github.com/mobilecoinfoundation/curve25519-dalek/pull/2 ?